install/kubernetes/Makefile

```diff
@@ -17,7 +17,7 @@ MIN_K8S_VERSION := "v$(MIN_K8S_MAJOR).$(MIN_K8S_MINOR).0"
 CILIUM_CHARTS := $(ROOT_DIR)/$(RELATIVE_DIR)/cilium
 CHART_FILE := "$(CILIUM_CHARTS)/Chart.yaml"
 
-VALUES_TMPL_FILE_NAME ?= *values.yaml.tmpl*
+VALUES_TMPL_FILE_NAME ?= *values.yaml.tmpl
```

Comment:
Removing the `*` as this will also match files left behind by rebase/merge conflicts (e.g. values.yaml.tmpl.orig). Hopefully that'll save the next person some time and confusion :D

Reply:
I believe these wildcards are there on purpose. Some distributions of Cilium use it to inject additional value templates during the build.